### ISSUE TITLE
Question about Loss Function settings for finetuning outdoor driving scene

### ISSUE THREAD

**Issue by sonic311** [Tsinghua-MARS-Lab/SLAM-Former#19]
I have a question for the authors.
I understand that you are also conducting research on training for driving scenes as a follow-up study. (SLAM-Former-infinity) I am curious if there are any tips regarding modifying the loss function or adjusting the loss function weights when fine-tuning with an outdoor driving dataset in the current SLAM-Former-training repository.
